packages/odyssey-storybook/src/components/customization/CustomTheme.stories.tsx

```diff
@@ -13,88 +13,79 @@
 import type { Story } from "@storybook/react";
 import {
   Button,
-  InputBase,
   FormControlLabel,
-  RadioGroup,
-  Radio,
   OdysseyThemeProvider,
+  Radio,
+  RadioGroup,
+  TextField,
+  ThemeOptions,
 } from "@okta/odyssey-react-mui";
+
 import CustomThemeMdx from "./CustomTheme.mdx";
 
 export default {
-  title: `Customization/components`,
-  component: Button,
+  title: "Customization/Components",
   parameters: {
     docs: {
       page: CustomThemeMdx,
     },
   },
 };
 
-const Template: Story = (props) => {
-  const customTheme = {
-    palette: {
-      primary: {
-        main: "rgba(233, 0, 0, 1)", // THIS IS A SAMPLE.
-      },
+const customTheme: ThemeOptions = {
+  palette: {
+    primary: {
+      main: "rgba(233, 0, 0, 1)", // THIS IS A SAMPLE. DO NOT USE!
     },
-  };
-
-  return (
-    <OdysseyThemeProvider customTheme={customTheme}>
-      <div>
-        {props.button && <Button variant="primary">Primary</Button>}
-        {props.input && (
-          <InputBase
-            autoComplete="name"
-            endAdornment={null}
-            id="demo-text-field"
-            inputProps={{
-              "aria-describedby": "textfield-hint textfield-error",
-            }}
-            startAdornment={null}
-            type="text"
-          />
-        )}
-        {props.radio && (
-          <RadioGroup
-            defaultValue="Lightspeed"
-            name="radio-buttons-group"
-            aria-describedby="radio-hint radio-error"
-          >
-            <FormControlLabel
-              value="Lightspeed"
-              control={<Radio />}
-              label="Lightspeed"
-            />
-            <FormControlLabel
-              value="Warp speed"
-              control={<Radio />}
-              label="Warp speed"
-            />
-            <FormControlLabel
-              value="Ludicrous speed"
-              control={<Radio />}
-              label="Ludicrous speed"
-            />
-          </RadioGroup>
-        )}
-      </div>
-    </OdysseyThemeProvider>
-  );
+  },
 };
 
-export const ButtonPrimary = Template.bind({});
-ButtonPrimary.args = {
-  button: true,
-};
+export const ButtonStory: Story = () => (
+  <OdysseyThemeProvider customTheme={customTheme}>
+    <div>
+      <Button variant="primary">Primary</Button>
+    </div>
+  </OdysseyThemeProvider>
+);
```

Comment:
`customTheme` should be different for each story to show how we can modify different things.

Make the button orange in this example.